metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/topology/ParserComponent.java

```diff
@@ -17,26 +17,23 @@
  */
 package org.apache.metron.parsers.topology;
 
-import java.io.Serializable;
-import org.apache.metron.parsers.bolt.WriterHandler;
 import org.apache.metron.parsers.interfaces.MessageFilter;
 import org.apache.metron.parsers.interfaces.MessageParser;
 import org.json.simple.JSONObject;
 
-public class ParserComponents implements Serializable {
+import java.io.Serializable;
+
+public class ParserComponent implements Serializable {
   private static final long serialVersionUID = 7880346740026374665L;
 
   private MessageParser<JSONObject> messageParser;
   private MessageFilter<JSONObject> filter;
-  private WriterHandler writer;
 
-  public ParserComponents(
+  public ParserComponent(
       MessageParser<JSONObject> messageParser,
-      MessageFilter<JSONObject> filter,
```

Comment:
WriterHandler was moved out of this class because it's handled by Storm.  MessageParser and MessageFilter is abstracted away from the streaming platform in this PR.